lib/commands/connect/import.js

```diff
@@ -18,13 +18,12 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/import';
-    fs.readFile(context.args.file, 'utf8', co.wrap(function* (err, data) {
-      data = JSON.parse(data);
-      let response = yield api.request(context.auth.password, 'POST', url, data);
-      console.log("Upload complete");
-      console.log(response.text);
+    let fName = context.args.file;
+    yield cli.action(`uploading ${fName}`, co(function* () {
+      let connection = yield api.withConnection(context, heroku);
+      let url = '/api/v3/connections/' + connection.id + '/actions/import';
+      let data = JSON.parse(fs.readFileSync(fName, 'utf8'));
```

Comment:
Given that export only writes to a file, it made sense to start by ignoring stdin. But it's probably easy enough to default to reading stdin if no filename is given. But also like the export case, that's probably for a different PR.